app/controllers/admin/users_controller.rb

```diff
@@ -2,7 +2,7 @@ module Admin
   class UsersController < ApplicationController
     before_filter :authorize_admin!
     def index
-      @users = User.all
+      @users = User.where(role: nil)
```

Comment:
Maybe users should have a default role? That way there is never a user with a `nil` role, and there is one less thing to worry about.